fe/fe-core/src/main/java/org/apache/doris/datasource/property/constants/S3Properties.java

```diff
@@ -334,6 +334,11 @@ public static Cloud.ObjectStoreInfoPB.Builder getObjStoreInfoPB(Map<String, Stri
             // S3 Provider properties should be case insensitive.
             builder.setProvider(Provider.valueOf(properties.get(S3Properties.PROVIDER).toUpperCase()));
         }
+
+        if (properties.containsKey(PropertyConverter.USE_PATH_STYLE)
```

Comment:
can it be modified by `alter vault` stmt?

Reply:
> can it be modified by `alter vault` stmt?

now, don't supported, maybe will support in the future